Dynamic require (#206)
rollup/plugins: packages/commonjs/README.md

Sometimes you have to leave require statements unconverted. Pass an array containing the IDs or an `id => boolean` function. Only use this option if you know what you're doing!

> ### `dynamicRequireTargets`

Reviewer: options should be in alphabetic order

Type: `String|Array[String]`<br>
Default: `[]`

> Some modules have dynamic `require` calls in them, or even circular dependencies (which are not handled well by EJS).  

Reviewer: ```suggestion
Some modules contain dynamic `require` calls, or require modules that contain circular dependencies, which are not handled well by [EJS](link to EJS info).  
```

What is EJS? Uncommon acronyms without links to info about them are generally unfriendly to new users. Please add a link to info about EJS.

Author: Wrong teminology.

> Including those modules as `dynamicRequireTargets` will simulate a CJS environment for them with support for dynamic and circular dependencies.  

Reviewer: Please don't use hard link breaks for paragraphs unless two or more lines are needed between them. Backspace so this line wraps naturally. We're big on getting our docs pristine here (even though we have a ways to go yet)

> Note that this might hardcode some paths from your PC in the final js file, so you might want to search & replace the base path of the root js file.

Reviewer: ```suggestion

_Note: This feature may result in some paths being rendered as absolute in the final bundle. That may require replacing ...<please explain more>._
```

I'm not quite sure what this note is supposed to tell the user they might have to do. I've suggested a change to the note for clearer initial language, but we nee to explain better to users what's up.
